src/rules/circularDependency.ts

```diff
@@ -227,8 +227,6 @@ function build(options: Options): RuleLogic {
 export default {
   id: 'circular-dependency',
   title: 'Circular package dependency',
-  // scope: //*[@command]
-  scope: 'command',
```

Comment:
For some rules, this PR changes the default scope to `root`, finding occurrences anywhere in the AppMap. This seems to me to make the most sense for code behaviors that are undesirable no matter where or in what context they occur.